src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -115,19 +115,18 @@ public Response createLibraryCardDaaRelation(
           return Response.status(Status.FORBIDDEN).build();
       }
       List<LibraryCard> libraryCards = libraryCardService.findLibraryCardsByUserId(userId);
-      Optional<LibraryCard> matchingCard = libraryCards.stream()
-          .filter(card -> card.getInstitutionId() == authedUser.getInstitutionId().intValue())
-          .findFirst();
-      if (matchingCard.isEmpty()) {
-        LibraryCard createdLc = libraryCardService.createLibraryCardForSigningOfficial(user, authedUser);
-        matchingCard = Optional.of(createdLc);
+      LibraryCard workingLibraryCard;
+      if (libraryCards.isEmpty()) {
+        workingLibraryCard = libraryCardService.createLibraryCardForSigningOfficial(user, authedUser);
+      } else {
+        workingLibraryCard = libraryCards.get(0);
```

Comment:
A future PR will restrict users to having a single library card. In the meantime, this should preserve the existing behavior.